test/bugs-reported/bug1.chpl

```diff
@@ -0,0 +1,100 @@
+
```

Comment:
Added this as a JIRA issue available here: https://chapel.atlassian.net/browse/CHAPEL-196
Added this long program to show the motivation as to why the programmer may need this. 
Will remove this bug as a "future" test as the JIRA issue has been created.